### ISSUE TITLE
UI: Topology host selection

### ISSUE THREAD

**Issue by milan-deepfence** [deepfence/ThreatMapper#1755]
1. Unable to start scan from other host due to Action button disabled while a scan is going on.

![Image](https://github.com/deepfence/ThreatMapper/assets/111341101/4ede85f0-8b49-4992-9e96-cd055dc8dac4)

2. Selection of multiple hosts are ignored and only first row host is sent for scan.

![Image](https://github.com/deepfence/ThreatMapper/assets/111341101/28cc4b96-4697-422e-8924-738d494cf40d)

**Comment by shyam-dev** [comment]
The metadata exists for servers that do not have an agent. Can that be used by the UI ?

**Comment by noboruma** [comment]
Traffic analysis is a ThreatStryker only feature - we should remove the column entirely from TM

**Comment by manV** [comment]
This is correct behavior, the hosts which are discovered by cloud scanner cannot be scanned, therefore if you only select hosts which are discovered by cloud scanner, the scan option is disabled. 

My suggestion is we should make the distinction clear. We can add a column for `Agent Running` Yes/No.
